func_tests/makefile

```diff
@@ -1,6 +1,6 @@
 CC = avr-gcc
 INCLUDES = -I../../include
-LIB = -L../../lib -lprintf_flt -lm -ladc -luart -lspi -lcan -lqueue -lstack -ltimer -lheartbeat -lconversions -lpex -ldac
+LIB = -L../../lib -lprintf_flt -lm -ladc -luart -lspi -lcan -lqueue -lstack -ltimer -lheartbeat -lconversions -lpex -ldac lwatchdog
```

Comment:
Should get rid of this change since the watchdog library isn't used in func_tests.

Reply:
Thanks for letting me know, I reverted that change.